test/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { sign, SignOptions } from "jsonwebtoken";
+import { configValues } from "../src/modules/configuration/index";
+
+export function generateJwtForUser(user: { id: number; shortId: string; uuid: string }, jwtOptions?: SignOptions) {
+  const payload = { id: user.id, uuid: user.uuid, shortId: user.shortId };
+  return sign(payload, configValues().auth.jwtSecret, jwtOptions);
+}
```

Comment:
Can we remove this method and use the existing 
https://github.com/across-protocol/scraper-api/blob/847f4304a754a16355d44cc075f5f9257dd23a76/src/modules/auth/services/auth.service.ts#L9
?

Reply:
I first tried to use this service in the tests like
```ts
const jwt = app.get(AuthService).generateJwtForUser(user)
```
but then I got an error
```
secretOrPrivateKey must have a value
```
and the service that I got looked like
```
AuthService {
  jwtService: JwtService {
    options: { secret: undefined, signOptions: [Object] },
    logger: Logger { context: 'JwtService', options: {} }
  }
}
```

Reply:
any idea why that is?

Reply:
I'll take a look and come back with an answer